go.sh

```diff
@@ -26,29 +26,35 @@ if [ "${LOCATIONS_CSV}" == "" ]; then
     LOCATIONS_CSV=/
 fi
 
+NAXSI_LISTEN_PORT="10418"
```

Comment:
Just pushed up a commit to address these changes. (Still using HTTPS though.)

Would be nice if someone wrote a test for this... \*hint\* \*hint\* \*nudge\* \*nudge\*.